ansible_collections/arista/avd/plugins/plugin_utils/eos_validate_state_utils/get_anta_results.py

```diff
@@ -168,7 +193,7 @@ def get_skipped_tests_from_tags(run_tags: tuple, skip_tags: tuple) -> list[dict]
     return result
 
 
-def generate_tests(device_name: str, hostvars: Mapping, skipped_tests: list[dict]) -> RawCatalogInput:
+def generate_tests(device_name: str, hostvars: Mapping, skipped_tests: list[dict], custom_catalog: dict | None = None) -> RawCatalogInput:
     """Create the test catalog in a dictionnary format generated from the AVD test classes.
 
     Test definitions are generated from the AVD structured_config for each AVD test classes and are merged together to create the final catalog.
```

Comment:
Add here something about custom_catalog
```suggestion
    Test definitions are generated from the AVD structured_config for each AVD test classes and are merged together with an optional custom_catalog to create the final catalog.
```

Reply:
Fixed in c490e182faf21a5c7f05d8c9fc38f181a875f708.